packages/router/__tests__/lazy-test.ts

```diff
@@ -212,6 +212,8 @@ describe("lazily loaded route modules", () => {
       expect(A.lazy.lazy.stub).toHaveBeenCalled();
 
       let lazyLoaderStub = jest.fn(() => "LAZY LOADER");
+      // Hmm, this feels like something's missing...
+      t.router.routes[0].lazy = undefined;
```

Comment:
Tests passed without this for me?  I removed it in my commit to change the 2 calls from `callLoaderOrAciton -> callLoaderOrActionImplementation`